### ISSUE TITLE
Keener - FEEDBACK Re: SA Billing: Merge Code Testing Environment Release 4/7/18 - FEEDBACK

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#799]
In GitLab by @kdjstudios on Apr 5, 2018, 15:20

**Submitted by:** Keener

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2018-04-05-87573/conversation

**Server:** Merge2

**Client/Site:** Keener

**Account:** NA

**Issue:** 

Due to the amount of feedback, I will have attached the full email; which includes my basic notes: [original_message__4_.html](/uploads/e7f6ac72a99ce61a54770706f23a7029/original_message__4_.html)

I will try my best to also list them below.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Apr 5, 2018, 15:21

- 1. Customer invites not working. - COMPLETED
 - I was able to explain that this is a just a testing server, and we removed the ability to send to external clients.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Apr 5, 2018, 15:23

- 2. Email sent had warning about spoofing
 - I explained to them that this was because we send from the email they have saved in the site settings, and the warning should go away once this is pushed to production. 

Will the warning go away?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Apr 5, 2018, 15:24

- 3. Billing Cycle Process:
 - The Create next billing cycle button is in middle not at the end; where it used to be?

What would cause this?

Also Is there a way to update the layout on those buttons, as they tend to always move around for me too. I do see the confusion here and it does not seem to look professional.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Apr 5, 2018, 15:31

- 4. Report issues: (Probably want to create a separate ticket for these items)
 - I walked through some of the reports, and there are a few issues such as: totals not matching across reports, totals not being close to correct, sometimes double what they should be, etc. I hinted that this is possibly due to when the 'snapshot' of data was copied over, but will have the support team confirm. 
 - Lots of the reports are including terminated accounts. - This is causing a lot of discrepancies and issues. - IT was mentioned that terminated accounts should not be in certain reports. Please see my notes on each one in the attachment on this ticket above. 
 - They need the ability to sort/filter reports
 - When first loading reports they were extremely slow, but after a few runs seemed to become faster. - What would cause this?

Please be sure to see my notes and the original concerns in the attachment on the main ticket for more information. During discussion with them I mentioned that report updates are going to be our next project, but we appreciate having their concerns now.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Apr 5, 2018, 15:36

- 5. Finalized Invoices
 - The invoices layout is still incorrect and confusing. Aging is way off. Please see attachment on main ticket.

I was able to discuss this with them and they suggested the following layout:

- Previous balance
- Payments
- Balance forward (previous balance - payments)
- space
- Current line charges
- space
- Total for invoice

Replace aging row at the bottom to be the following:
- 30,60,90,this invoice total, then total due

And of course; to have the aging at the bottom calculate correctly.

**Comment by kdjstudios** [comment]
In GitLab by @tim.traylor on Apr 6, 2018, 06:08

@kdjstudios Hi Kyle, I want to make sure we're on the same page here and that I'm understanding correctly.  Based on my discussion with the Aavaz team this AM, these issue exist in production right now.  SO they're really just asking for changes to the system and not reporting an error on Merge.

Also, the invoice aging is related to the known problem with the old calculations and will only be corrected by a manual adjustment of the previous invoice buckets.  This is the email that needs to go out to the clients today.  Thx, Tim

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Apr 6, 2018, 09:15

@tim.traylor @smasih 

Thank you for the update, You are correct all issues are known issues, and continue to be known issues. None of them appear to be issues specific to Merge. Unless the miscalculations mentioned in all the reports are Merge related issues, we would need the Aavaz team to check and confirm that. 

Oh.. Except for the email spoofing, that is the first time I had heard someone see or complain about that I believe.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Apr 6, 2018, 12:01

Hi [Kyle] (@kdjstudios ),   

Yeah, we too are not sure about the case. Can you please explain that where this happened? It would really help to dig into the case with more details.

Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Apr 6, 2018, 12:44

@pchaudhary 

Please see the original email attachment in the main ticket here. Jannemieke does a wonderful Job describing the issue with each report. You should simply be able to run those same reports to verify. 

Marking for discussion.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on May 28, 2018, 01:47

- Spoofing warning. Anyway to remove [+repeation of payment confirmation box - Fixed+]

- Create next billing cycle button is in middle not at the end. `re-structure billing cycle page 1.5hr`

- Keener cali, any way to hide/remove `site, accounts and row in billing cycle row - 30m`

- Reports are still slow. But got faster. `??`

- Revenue Analysis Report  [-need discussion to understand the request-]  
    a.       This has some issues.    
    b.      Should filter out terminated accounts…filter/sort option      
    c.       Late charges are off on the late charges, - due to snap shot    

- Side by side [-need discussion to understand the request-]  
    a.      Period charges not showing    
    b.      Difference in comparing other reports    
    c.      Late charges not showing    
    d.      Holiday charges show under debits, but need their own category.    

- Review Draft invoices report `estimate - 1-2hr`  
    a.       Total is double what the normal, - possible due to snapshot    
    b.      Sort/filter out terminated accounts.    

- Aging report `estimate - 2hr`  
    a.       Sort filter out terminated accounts.    
    b.      Columns F through J do not equal.. Miscalculation in possible totals.    

- Accounts Receivables Reports [-need discussion to understand the request-]  
    a.       Current is off then the aging and what was billed.      
      i.      Possible matches the Review draft invoice, double totals.      

- Finalized invoices,          
* Aging line is incorrect [+Need an account from them for investigating+]  
    
   * Previous is correct        
   * Invoice current is correct      
   * Current in aging line does not match the current invoice.      
   * Possible the rest of them are incorrect.    
   * Balance forward, payments, this invoice then 30,60,90,then total due      

* Layout [-Need more discussion on this-]     

  * Previous balance      
  * Payment should show up at top    
  * Balance forward    
  * space    
  * Current charges    
  * Total for invoice    
  * At bottom show the 30,60,90,and this invoice, along with total due    

- Service code summary  [-discussion required-]  
* Matches, but where are the period charges?    
 * These are on the review analysis report why      
* Matches except for the late charges?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 29, 2018, 12:06

Please bring this up in discussion on Thursday.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 21, 2018, 08:55

@smasih @pchaudhary 

Any updates on this?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 29, 2019, 14:11

Marking for review with @tim.traylor

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on May 23, 2019, 01:18

@kdjstudios 

Can we close this?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 23, 2019, 08:40

@smasih @amishra @pchaudhary @tim.traylor 

Sorry, I forgot to hit the comment button, this is what I had below:

Follow up on Keener Cali site (hide/remove) will they need any access to the data or is this for a complete removal. We will still want to keep a long term backup of this.  

Revenue Analysis Report - Should filter out terminated accounts…filter/sort option


Period charges not showing - what is this referring too?

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on May 31, 2019, 06:56

@kdjstudios 

Sorry, but we are not able to get this. Could you please elaborate if we have to do anything or we can mark this for discussion.

Thanks

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 3, 2019, 15:01

@pchaudhary 

This is labeled ForAN, we will update the labels once it is ready for the Aavaz team.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Jul 9, 2019, 24:39

removed milestone

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 22, 2019, 12:22

@tim.traylor 

I believe we can close this ticket as it is a Keener request. and most of the requests have already been resolved.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 29, 2019, 09:35

closed
